web_app/dist/title.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link href="https://fonts.googleapis.com/css?family=Raleway" rel="stylesheet">
+        <link rel="icon" type="images/png" href="images/icon.png" />
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>TaskViewer</title>
+        <meta name="description" content="Schedule tasks, without the clutter">
+        <link rel="stylesheet" href="title.css">
+    </head>
+
+    <body>
+    
+        <div class="title-text-container absolute-center">
+            <h1 id="title" class="title-page-font title-page-spacing">TaskViewer</h1>
+            <h2 id="tagline" class="title-page-font title-page-spacing">Schedule tasks, without the clutter.</h2>
+        </div>
+
+        <div class="logo-center absolute-center">
+            <img id="logo" src="images/logo.png" alt="TaskViewer Logo">
```

Comment:
Is the `logo` id styled in the CSS? I'm not quite sure why the id is necessary

Reply:
fixed.